src/modules/Experiences/ExperiencesList/DeleteExperience.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import withStyles from '@material-ui/core/styles/withStyles';
+
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+
+const DeleteExperience = ({ experience, onDelete, classes, handleCancel }) => {
+  const experienceName = `${experience.host} - ${experience.title}`;
```

Comment:
Like I mentioned earlier today, this dash should probably be an `&mdash;` rather than a hyphen.
This is very minor but also a very quick fix 
For more info: https://getitwriteonline.com/articles/en-dashes-em-dashes/

Reply:
Oh, also it might be worth thinking about what happens here if `host` or `title` is empty/undefined/null—right now that _should_ be impossible but ideally this experience either (a) explicitly notes that it is dependent on host and title (e.g. by taking them as explicit arguments rather than just part of experience), or by (b) having a fallback option if one or both are null.